addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -301,11 +301,9 @@ export class WebsitePreview extends Component {
         if (!this.backendUrl) {
             this.backendUrl = routeToUrl(this.router.current);
         }
-        const currentUrl = new URL(this.iframe.el.contentDocument.location.href);
-        currentUrl.pathname = `/@${currentUrl.pathname}`;
-        this.currentTitle = this.iframe.el.contentDocument.title;
```

Comment:
Removing a variable from `this` in stable in general I would ask to change... but let's not overthink this one :)